package/netsurf/package

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+# Copyright (c) 2021 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(netsurf)
+pkgdesc="Lightweight web browser"
+url=https://github.com/alex0809/netsurf-reMarkable
+pkgver=0.3.3-1
+timestamp=2021-04-12T15:14+00:00
+maintainer="Alex Friesenhahn <rm-dev@alexfriesenhahn.de>"
+archs=(rmall)
+license=MIT
+section="readers"
+installdepends=(
+    dejavu-fonts-ttf-DejaVuSans
+    dejavu-fonts-ttf-DejaVuSans-Bold
+    dejavu-fonts-ttf-DejaVuSans-BoldOblique
+    dejavu-fonts-ttf-DejaVuSans-Oblique
+    dejavu-fonts-ttf-DejaVuSerif
+    dejavu-fonts-ttf-DejaVuSerif-Bold
+    dejavu-fonts-ttf-DejaVuSerif-Italic
+    dejavu-fonts-ttf-DejaVuSansMono
+    dejavu-fonts-ttf-DejaVuSansMono-Bold
+)
+image=base:v1.5
+
+source=(
+    https://github.com/alex0809/netsurf-reMarkable/archive/refs/tags/v0.3.3.tar.gz
+    netsurf.draft
+)
+sha256sums=(
+    9d6bb15af25604a390a45d665445b53b8b2530f01240054dbe1a7c306f1b90ac
+    SKIP
+)
+
+build() {
+    scripts/install_dependencies.sh
```

Comment:
@matteodelabre thoughts on this?

Reply:
It would be better to take advantage of Toltec’s dependency management capabilities to handle this.

* Use the `makedepends=(build:a build:b …)` syntax to specify Debian packages that need to be installed before compiling. See [the docs](https://github.com/toltec-dev/toltec/blob/testing/docs/package.md#makedepends) for more info on this.
* Add new recipes for libraries on which this package depends (libcurl, libfreetype, libjpeg-turbo—from what I can see). See what was done with [libdlib](https://github.com/toltec-dev/toltec/blob/testing/package/libdlib/package) and [recrossable](https://github.com/toltec-dev/toltec/blob/testing/package/recrossable/package) for example.
* Note that a CMake toolchain file is already included in the Toltec build toolchain, so you don’t need to recreate it. See [vnsee](https://github.com/toltec-dev/toltec/blob/testing/package/vnsee/package) for an example of a project that uses CMake.

Reply:
OK thanks for the links, @matteodelabre.
Would you prefer to handle these as separate pull requests each or with this one?

Also, I'll be happy to create the new packages for these dependencies, but I'm not so comfortable with setting myself as maintainer for them because I can't foresee how much free time I have available for this longer-term.

Reply:
I believe you can leave the maintainer blank and one of us will pick it up when we got around to approving it.

Reply:
> Add new recipes for libraries on which this package depends (libcurl, libfreetype, libjpeg-turbo—from what I can see). See what was done with libdlib and recrossable for example.

>  Would you prefer to handle the new recipes as separate pull requests each or with this one?

i think this (making new packages for each dep) should only be done if alex has time + cycle. we should first review the package, then refactor the dependencies in a separate PR, so this PR doesn't get held up.

Reply:
@raisjn For sure, all I'm waiting on is moving the build dependencies to `makedepends`. Which looks like it got committed a little while ago, so now I'm waiting for a build with it to test.